encryption.rst

```diff
@@ -223,3 +360,24 @@ Running using an environment variable
 .. code::
 
    $ {ENVPREFIX}_ENCRYPTION_PEM_PATH=rsa_pri.pem {command} run encrypted.sif
+
+.. note:: 
```

Comment:
I think that the note probably will not include all the following ..code sections you intended to include.  So probably instead of making a note here it should be a separate section.

Reply:
thanks for the comments. reflected in this commit https://github.com/apptainer/apptainer-userdocs/pull/179/commits/c7f08f6f219f7edc65f6a48fdae506166718c9e3